public/components/trace_analytics/components/common/plots/service_map.tsx

```diff
@@ -302,11 +420,13 @@ export function ServiceMap({
               <div style={{ position: 'relative' }}>
                 {items?.graph && (
                   <Graph
+                    key={graphKey}
                     graph={items.graph}
                     options={options}
                     events={events}
                     getNetwork={(networkInstance: any) => {
```

Comment:
does networkInstance have a type that can be set here? making sure that this has a type might also require setZoomLimits to set the input variable type as well, so ideally that function should have a type for networkInstance

Reply:
I think this would require a additional dependency of @types/vis-network